### DISCUSSION TITLE
Stuck on SubmitCommitAggregate

### DISCUSSION THREAD

**Discussion by btclinux** [filecoin-project/lotus#9195] [Storage Provider] [answered]
I use single machine setup. Lotus v1.17.0. I run "lotus-miner sectors pledge" and now process stuck.

~$ lotus-miner sectors list
ID  State                  OnChain  Active  Expiration  Deals
4   SubmitCommitAggregate  NO       NO      n/a         CC

~$ lotus-miner sectors status -log 4
...
10.     2022-08-20 00:35:31 +0500 +05:  [event;sealing.SectorSubmitCommitAggregate]     {"User":{}}

Miner do nothing at this stage. What I have to do to continue sealing process?

**Comment by TippyFlitsUK** [comment, marked as answer]
Hi @btclinux :wave:

This sector is not stuck, it has been batched and this is the intended behavior.

You can read more about batching [here](https://lotus.filecoin.io/storage-providers/advanced-configurations/sealing/#provecommitaggregate).

You can force this sector to commit by running `lotus-miner sectors batching commit --publish-now`.
